projects/ng2-date-picker/src/lib/day-calendar/day-calendar.component.ts

```diff
@@ -219,6 +225,31 @@ export class DayCalendarComponent implements OnInit, OnChanges, ControlValueAcce
       return;
     }
 
+    if (this.dateRangePicker) {
+      this.dateRange ||= { from: null, to: null };
+
+      if (this.dateRange.from && this.dateRange.to) {
+        this.dateRange = { from: null, to: null };
+      };
+
+      if (!this.dateRange.from) {
+        this.dateRange.from = day.date;
+        return;
+      }
+
+      if (day.date.isBefore(this.dateRange.from)) {
+        this.dateRange.from = day.date;
+        return;
+      }
+
+      this.dateRange.to = day.date;
+      this.onDateRangeSelect.emit(this.dateRange);
+      // eslint-disable-next-line no-console
```

Comment:
please remove

Reply:
What's wrong here? Thank you for review.

Reply:
Oh okey, you want to remove eslint-disable line. I currently using this line for testing.